Library/Formula/emacs.rb

```diff
@@ -39,6 +40,7 @@ class Emacs < Formula
   depends_on "imagemagick" => :optional
   depends_on "mailutils" => :optional
   depends_on "glib" => :optional
+  depends_on "libxml2" => :optional
```

Comment:
We don't support Linuxbrew here; Linux formulae changes should be made in Linuxbrew, sorry.